### PR TITLE
Implement Agency API according to OpenAPI defs

### PR DESCRIPTION
**Pull request by wingsofovnia**

### CODE REVIEW

File: backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/v2/mapping/MappingException.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    public MappingException(String message) {
+        super(message);
+    }
+
+    public MappingException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    protected MappingException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
```
Comment (glasierr) [inline backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/v2/mapping/MappingException.java]: Why protected?

File: backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/util/Sorts.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+     */
+    public static Sort parse(String[] sortCsv) {
+        return stream(sortCsv).map(Sorts::parseSortOrder).collect(collectingAndThen(toList(), Sort::new));
+    }
+
+    private static Sort.Order parseSortOrder(String sortCsv) {
+        val sortPropDir = sortCsv.split(SEARCH_PROP_DELIMITER);
+
+        if (sortPropDir.length == 0)
+            throw new IllegalArgumentException("String doest match prop" + SEARCH_PROP_DELIMITER + "direction pattern");
```
Comment (glasierr) [inline backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/util/Sorts.java:56]: Concatenated string will look strange. Probably should add whitespaces around variable and wrap it with single quotes

**Review by glasierr — APPROVED** [review]

File: backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/util/Sorts.java (hunk 2 above)
Comment (wingsofovnia) [reply backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/util/Sorts.java:56]: no, this is the exact format supported: e.g. `?sort=name,desc` or `?sort=age,asc`

Comment (glasierr) [reply backend/backend-application/src/main/java/eu/socialedge/hermes/backend/application/api/util/Sorts.java:56]: Oh, it wasn't clear without single quotes :)

**Pull request merged**